content/en/docs/concepts/workloads/pods/user-namespaces.md

```diff
@@ -46,7 +46,21 @@ tmpfs, Secrets use a tmpfs, etc.)
 Some popular filesystems that support idmap mounts in Linux 6.3 are: btrfs,
 ext4, xfs, fat, tmpfs, overlayfs.
 
-In addition, support is needed in the 
+In addition, support is needed in the container runtime and in the OCI container
+runtime.
+
+The OCI container runtime requirements are as follows:
+
+ * [crun][crun]: version 1.9 or greater (1.13+ recommended)
+
+Please note that [runc][runc] is the default OCI container runtime in several
+Kubernetes distributions. Version `1.1.z` of runc doesn't support all the
+features needed by the Kubernetes implementation. If there is a newer release of
+runc than 1.1 available for use, check its documentation and release notes for
+compatibility (look for idmap mounts support in particular, as that is the
+missing feature).
```

Comment:
Try this:

```suggestion

### Container runtime compatibility

To use user namespaces with Kubernetes, you need to use an OCI container runtime that
provides support for idmapped mounts; check the list below or verify with your chosen
runtime that it offers the features that Kubernetes needs.

{{% thirdparty-content single="true" %}}

Here is a list of OCI container runtimes that provide the features required for
user namespaces in Linux pods:

* [crun][crun]: version 1.9 or greater
  * Using version 1.13 or later provides the best overall experience

```

We very very rarely provide lists of software that isn't compatible (it could be a long list). There's an exception for containerd specifically because we previously did list it as compatible.

Reply:
See https://github.com/kubernetes/website/pull/45178#issuecomment-1994316624

Reply:
I've seen it, I can't see why it is relevant. It says:

> I don't know of any place where Kubernetes hard codes runc as a kubelet default, or Pod API default, or RuntimeClass default; if is is, though, I can raise an issue (we should document this defaulting).
> 
>So far as I know, runc is very popular but Kubernetes is agnostic about which OCI runtime you use.

I didn't claim Kubernetes hardcodes anything about runc, so I really don't understand why you say that.

My point is: it is a widely used OCI runtime, therefore it is relevant information I want to provide to users.

Reply:
@sftim this text was changed several times, last time by @drewhagen suggestions. Do you still think this suggestion is better than what is currently written?

Reply:
Let's mark this done and re-review